### PR TITLE
Add timer and update validation every minute

### PR DESCRIPTION
**Pull request by Erbidium**

### CODE REVIEW

**Review by Pavliuk13 — APPROVED** [review]

File: frontend/src/app/modules/exclusion-dates/exclusion-dates-picker/exclusion-dates-picker.component.ts [hunk 1]
```diff
@@ -30,6 +31,12 @@ export class ExclusionDatesPickerComponent extends BaseComponent implements OnIn
 
     ngOnInit() {
         this.formGroup = this.getDefaultFormGroup();
+        const startDelayMilliseconds = (60 - new Date().getSeconds()) * 1000;
+        const minuteMilliseconds = 60000;
+
+        timer(startDelayMilliseconds, minuteMilliseconds)
+            .pipe(this.untilThis)
+            .subscribe(() => this.updateStartRangeValidators());
```
Comment (tatianahutii) [inline frontend/src/app/modules/exclusion-dates/exclusion-dates-picker/exclusion-dates-picker.component.ts]: Why do we need this?

**Review by tatianahutii — CHANGES_REQUESTED** [review]

File: frontend/src/app/modules/exclusion-dates/exclusion-dates-picker/exclusion-dates-picker.component.ts (hunk 1 above)
Comment (Erbidium) [reply frontend/src/app/modules/exclusion-dates/exclusion-dates-picker/exclusion-dates-picker.component.ts]: The reason is that when user selects current date as exclusion date and enters time range, then all time ranges which start before the current moment of time are considered to be invalid. This peace of code makes sure that all entered, but not saved time ranges are valid in current moment of time

Comment (tatianahutii) [reply frontend/src/app/modules/exclusion-dates/exclusion-dates-picker/exclusion-dates-picker.component.ts]: But why is this time range invalid?

Comment (Erbidium) [reply frontend/src/app/modules/exclusion-dates/exclusion-dates-picker/exclusion-dates-picker.component.ts]: I thought each time range should be completely in the future. I can rework it by applying this validation to the end of the time range to check if an entire time range is not in the past

Comment (Erbidium) [reply frontend/src/app/modules/exclusion-dates/exclusion-dates-picker/exclusion-dates-picker.component.ts]: Fixed

**Review by tatianahutii — APPROVED** [review]

**Review by apofis96 — APPROVED** [review]

**Pull request merged**